distributed/cli/dask_worker.py

```diff
@@ -26,214 +26,237 @@
 
 logger = logging.getLogger("distributed.dask_worker")
 
-
 pem_file_option_type = click.Path(exists=True, resolve_path=True)
 
+worker_options = [
+    click.Option(
+        ["--tls-ca-file"],
+        type=pem_file_option_type,
+        default=None,
+        help="CA cert(s) file for TLS (in PEM format)",
+    ),
+    click.Option(
+        ["--tls-cert"],
+        type=pem_file_option_type,
+        default=None,
+        help="certificate file for TLS (in PEM format)",
+    ),
+    click.Option(
+        ["--tls-key"],
+        type=pem_file_option_type,
+        default=None,
+        help="private key file for TLS (in PEM format)",
+    ),
+    click.Option(
+        ["--worker-port"],
+        default=None,
+        help="Serving computation port, defaults to random. "
+        "When creating multiple workers with --nprocs, a sequential range of "
+        "worker ports may be used by specifying the first and last available "
+        "ports like <first-port>:<last-port>. For example, --worker-port=3000:3026 "
+        "will use ports 3000, 3001, ..., 3025, 3026.",
+    ),
+    click.Option(
+        ["--nanny-port"],
+        default=None,
+        help="Serving nanny port, defaults to random. "
+        "When creating multiple nannies with --nprocs, a sequential range of "
+        "nanny ports may be used by specifying the first and last available "
+        "ports like <first-port>:<last-port>. For example, --nanny-port=3000:3026 "
+        "will use ports 3000, 3001, ..., 3025, 3026.",
+    ),
+    click.Option(
+        ["--bokeh-port"],
+        type=int,
+        default=None,
+        help="Deprecated.  See --dashboard-address",
```

Comment:
We probably don't need to add deprecated options to `dask-ssh` (since they didn't exist on that command before).

Reply:
Add the following to ignore options with `deprecated` in their help:
```python
    if hasattr(opt, "help") and "deprecated" in opt.help.lower():
        continue
```